includes/admin/class-admin.php

```diff
@@ -1133,6 +1135,15 @@ public function ajax_dismiss_notification_server_requirements() {
             set_transient( 'wpbdp_server_requirements_warning_dismissed', true, WEEK_IN_SECONDS );
         }
 
+		/**
+		 * Set fontawesome notice as dismissed.
+		 *
+		 * @since x.x
+		 */
+		public function ajax_dismiss_notification_fontawesome_dismissed() {
+			wpbdp_set_option( 'enqueue-fontawesome-styles', false );
```

Comment:
We still need nonce and permissions checks here since it's an ajax call.

Reply:
Nonce are handled in the function `ajax_dismiss_notification` that is used on the plugins dismiss messages. I'll add permission check on it